src/Webinex.Calendar/Calendar.cs

```diff
@@ -96,14 +96,29 @@ async Task<RecurrentEvent<TData>[]> IRecurrentEventCalendarInstance<TData>.GetMa
         return result.Select(x => x.ToRecurrentEvent()).ToArray();
     }
 
+    async Task<RecurrentEvent<TData>[]> IRecurrentEventCalendarInstance<TData>.GetAllAsync(FilterRule filter)
+    {
+        filter = filter.Replace(new RecurrentEventFilterRuleVisitor());
+
+        var local = Filter(_dbContext.Events.Local.AsQueryable()).ToArray();
```

Comment:
it doesn't make any sense, if you're not using AsNoTracking on db query - it will return same instances

Reply:
here I'm using `_dbContext.Events.Local`. If we add any new row and not SaveChanges it will be in Local, but not in db